support/doc/dependencies.md

```diff
@@ -20,10 +20,17 @@
 ```
 $ sudo apt update
 $ sudo apt install nginx ffmpeg postgresql openssl g++ make redis-server git
-$ ffmpeg -version # Should be >= 3.x 
+$ ffmpeg -version # Should be >= 3.x
 $ g++ -v # Should be >= 5.x
 ```
 
+if you still have a 2.x ffmpeg version on ubuntu-server :
```

Comment:
"If you still have a 2.x version of `ffmpeg` on Ubuntu:"

Reply:
I fixed it myself: https://github.com/Chocobozzz/PeerTube/commit/30274d67ebd47a5cbbd57ea1cb14fb904e5e4445

Thanks for the review @rigelk